schema/bom-1.7.xsd

```diff
@@ -4020,7 +4020,7 @@ limitations under the License.
             </xs:documentation>
         </xs:annotation>
         <xs:sequence>
-            <xs:element name="description" minOccurs="0" maxOccurs="1">
+            <xs:element name="description" type="xs:string" minOccurs="0" maxOccurs="1">
```

Comment:
in JSON its a strings.
in protobug it is string.

conclusion: was intended to be string in XML, too